internal/repository/mongo.go

```diff
@@ -0,0 +1,62 @@
+package repository
+
+import (
+	"context"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+
+	"github.com/andreipimenov/golang-training-2021/internal/model"
+)
+
+type Client struct {
+	*mongo.Client
```

Comment:
I don't understand why the mongo.Client should be here.

Reply:
IMHO mongo.Client being here keeps as much db close in one file as possible, which leads to cleaner architecture. There might be a cleaner solution, though..

Reply:
Interesting. I understood. Thanks